### PR TITLE
Remove stand arrow from Lavaland

### PR DESCRIPTION
**Pull request by KingTerrytheTerrible**
Too many miners keep rolling miner solely to search for it and nothing else and suicide when it's not there, taking up slots and shit. It just causes cancer for no reason when we already have stand-givers on Lavaland which are harder to get

### CODE REVIEW

**Comment by karmaisblackandbluepilled** [comment]
+1 fuck the stando arrow

**Comment by danxnader** [comment]
-1 fuck off

**Comment by MrEpicSax** [comment]
-1 Obviously a problem with player attitude. What, you gonna start take the toys away from the rest of us because little Timmy keeps shoving transformers up his ass? The arrow isn't the issue. A certain someone has a grudge.

If someone wants to play miner in the hopes of getting the arrow that's perfectly fine. If someone consistently takes up a valuable slot only to commit suicide when things don't go their way, without first supplying the station with necessary resources mind you, then yeah you might have a problem and it's that the person hasn't been job banned.

**Comment by KingTerrytheTerrible** [comment]
looks like we found the miners who roll only for stand arrows

**Comment by KingTerrytheTerrible** [comment]
sweaty your opinions dont matter =)

**Comment by MrEpicSax** [comment]
:^(

**Comment by FantasticFwoosh** [comment]
Miners shouldnt be so off task anyway, perhaps a time/progress lock should be in place in that the door to the ruin with inpenetrable walls wont open until a lavaland boss has been slain.

Trading one problem for another but atleast it requires effort.

**Comment by zamolxius** [comment]
+1 fuck stando arrow. it needs a better vehicle to enter the game than this

**Comment by TrickyDickRichard** [comment]
+1 but just bring back asteroid

**Comment by yoyobatty** [comment]
+1 stands are also cancer as fuck, it takes up majority of any antag roles nowadays. Literally everyone uses them since it saves from getting absolutely fucked by having a backup friend to assist. It's the new meta and it's lame as fuck.

**Comment by Pyko1** [comment]
+1 oh it really did take you guys this long to realize how bad idea it was?

**Comment by Manly-Dwarf** [comment]
This is the inherent problem with Lavaland's loot system.
Reminder that Lavaland gives out antag to miners for free just because they're miners.

**Comment by Toxici11i** [comment]
> This is the inherent problem with Lavaland's loot system.
> Reminder that Lavaland gives out antag to miners for free just because they're miners.

yeah but like the entire reason they do that is so people play miner, get resources and essentially save the station, the main issue is are people that play miner that suicide/ghost/afk if they dont roll antag or stand arrow. Those are the people who antag/stand fish and should be punished, not legitimate players

**Comment by MrEpicSax** [comment]
> 
> 
> Miners shouldnt be so off task anyway, perhaps a time/progress lock should be in place in that the door to the ruin with inpenetrable walls wont open until a lavaland boss has been slain.
> 
> Trading one problem for another but atleast it requires effort.

> 
> 
> > This is the inherent problem with Lavaland's loot system.
> > Reminder that Lavaland gives out antag to miners for free just because they're miners.
> 
> yeah but like the entire reason they do that is so people play miner, get resources and essentially save the station, the main issue is are people that play miner that suicide/ghost/afk if they dont roll antag or stand arrow. Those are the people who antag/stand fish and should be punished, not legitimate players


Giving it a prerequisite is a decent idea and I 100% agree with Toxic. However, Yoyobatty has a fair point about the meta being stale which the arrow contributes to in addition to having a 50% chance of taking the miner (and who knows else) out of the round even if they do find it. On top of that, it's unlikely it would stay forever removed either since that would be an unnecessary waste. I guess if I were to put my bias towards the epic jojo item aside for a moment then I could say that it deserved to be removed, particularly given that so many of you seem to dislike it being here. In saying that, this wouldn't exactly deal with the root issue/s either. So I hope this goes through on some level, and I also hope it eventually returns in some way should it get removed.

TL;DR +1, I changed my mind.

**Comment by Manly-Dwarf** [comment]
ah yes let me go get those minerals to save the station, and my reward might be a quick little antag round
nah just kiddin gonna go rush bubblegum and make myself antag within five minutes
thanks lavaland

**Comment by AnthonyJNgo** [comment]
Yeah +1, everyone else has already given the good points on why this should be approved.
We can look into adding more ruins or something later. Not a huge thing that's being removed in any case.

**Comment by MrEpicSax** [comment]
> 
> 
> ah yes let me go get those minerals to save the station, and my reward might be a quick little antag round
> nah just kiddin gonna go rush bubblegum and make myself antag within five minutes
> thanks lavaland

You're not wrong. But you're not right either.

**Comment by karmaisblackandbluepilled** [comment]
why's travis shidding itself

**Comment by KingTerrytheTerrible** [comment]
BYOND down and DreamChecker out of date

**Comment by Toxici11i** [comment]
> why's travis shidding itself

its travis

**Comment by mkikt4743** [comment]
aight cool just make it so there is a prereq like defeating 3 bosses +1

**Comment by DaydreamPuppeh** [comment]
When can we delete lavaland and bring back the asteroid

**Comment by diegoxt45** [comment]
how about we make stand arrows actually rare again and like make em meteor only

**Comment by danhibiki** [comment]
+1 stands on lava land were a mistake, there is already enough goodies on lava land to not need something that is an antag item to be easily acquired

**Comment by QuarianCommando** [comment]
+1 considering the one and only downside to being a miner is the fact that nobody is going to come and save your ass if you fuck up, and the stand arrow give you someone to save your ass if you fuck up.

**Comment by Maroonenahmoron** [comment]
+1, this thing is too easy to get. Make the ruin harder to access/go through or remove it if that's just too much.

**Comment by danhibiki** [comment]
This has majority approval from headmins and is ready to be merged after the map errors are fixed

**Comment by AnthonyJNgo** [comment]
Still waiting on the merge conflicts to be resolved.

**Comment by yoyobatty** [comment]
Address changes or it gets CLOSED!1

**Comment by AnthonyJNgo** [comment]
This is probably not going to be addressed. I'll handle this myself, just going to leave this PR up until I get around to doing it, then I will close this one and reference it in my new PR.

**Comment by carlopault** [comment]
The arrow should be obtainable in another way then. What if you have to assemble the arrow from collecting 3 parts of it: Stand Arrow Head, Stand Arrow Shaft, Stand Arrow Fletching. Maybe have the Arrow Head spawn in maint, the Shaft spawn on Lavaland, and the Fletching can be found at another negotiable location.

**Comment by KingTerrytheTerrible** [comment]
i don't exactly get notifications when I'm banned from the discord

**Comment by AnthonyJNgo** [comment]
> 
> 
> i don't exactly get notifications when I'm banned from the discord

I didn't even know you were still on Github LOL. How can we reliably contact you? Tell steamport or something he can let me know.

**Comment by KingTerrytheTerrible** [comment]
> 
> 
> > i don't exactly get notifications when I'm banned from the discord
> 
> I didn't even know you were still on Github LOL. How can we reliably contact you? Tell steamport or something he can let me know.

reliably? email or matrix

**Comment by ErwinSnowden** [comment]
> The arrow should be obtainable in another way then. What if you have to assemble the arrow from collecting 3 parts of it: Stand Arrow Head, Stand Arrow Shaft, Stand Arrow Fletching. Maybe have the Arrow Head spawn in maint, the Shaft spawn on Lavaland, and the Fletching can be found at another negotiable location.

This doesn't make sense because the arrow head is the actual functional artifact, also this is Hippie, if we're going to remove stuff because staff won't handle it in-game we're just becoming a shitty /tg/ offshoot. Don't remove fun because you don't like it, instead rework it so it's locked behind something that encourages people to do it to get their funnies and get mats for the station, like say, making it so you have to put 20~ refined diamonds into a receptacle or something in order to open up the door to the stand arrow ruin, it'd also stop the metarush in cases like nukies (where it can completely de-balance a round or just landslide it for the nukies, since 50% of the station dies because arrow destroys your very soul.)

tl;dr removing shit is shit, rework shit, get it done, don't remove content.

also on the note of that, if we're removing problematic shit that we just don't like, we should remove the high-frequency blade since you know, fun is bad.

**Comment by Toxici11i** [comment]
> > The arrow should be obtainable in another way then. What if you have to assemble the arrow from collecting 3 parts of it: Stand Arrow Head, Stand Arrow Shaft, Stand Arrow Fletching. Maybe have the Arrow Head spawn in maint, the Shaft spawn on Lavaland, and the Fletching can be found at another negotiable location.
> 
> This doesn't make sense because the arrow head is the actual functional artifact, also this is Hippie, if we're going to remove stuff because staff won't handle it in-game we're just becoming a shitty /tg/ offshoot. Don't remove fun because you don't like it, instead rework it so it's locked behind something that encourages people to do it to get their funnies and get mats for the station, like say, making it so you have to put 20~ refined diamonds into a receptacle or something in order to open up the door to the stand arrow ruin, it'd also stop the metarush in cases like nukies (where it can completely de-balance a round or just landslide it for the nukies, since 50% of the station dies because arrow destroys your very soul.)
> 
> tl;dr removing shit is shit, rework shit, get it done, don't remove content.
> 
> also on the note of that, if we're removing problematic shit that we just don't like, we should remove the high-frequency blade since you know, fun is bad.

the difference is the HF blade is a terrible item that still works with stealth box + hf and if you get discount combine with throwing weapon.

Removing a stand arrow because "miners having a reason to mine" and claiming it "gives miner free antag" is wrong. being a stand user does not give you, or your stand a reason to act like an antagonist, as stated in the rules

**Comment by MrEpicSax** [comment]
I shit my pants :^(
